modules/local/process/bwamem2_mem.nf

```diff
@@ -1,19 +1,28 @@
+include { initOptions; saveFiles; getSoftwareName } from './../../nf-core/software/functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+environment = params.enable_conda ? "bioconda::bwa-mem2=2.0 bioconda::samtools=1.10" : null
```

Comment:
FYI: BWA-Mem2 was updated a couple of days ago to a new release 2.1. It is in conda already, but as far as I saw the biocontainers are not updated yet. Apparently, the new release reduces index size quiet a bit :)

Reply:
Good to know, I'll keep a look at the containers.